tests/lib/timelib.py

```diff
@@ -11,120 +11,18 @@
 
 import pytz  # pylint: disable=wrong-import-order
 
+from tests import test_lib as shared_test_lib
 
-class TimeLibTest(unittest.TestCase):
-  """Tests for timestamp."""
-
-  def testCopyFromString(self):
-    """Tests the CopyFromString function."""
-    timestamp = timelib.Timestamp.CopyFromString('2012-06-27')
-    expected_timestamp = 1340755200000000
-    self.assertEqual(timestamp, expected_timestamp)
-
-    with self.assertRaises(ValueError):
```

Comment:
Don't we still want to test that bad timestamps will raise expectedly?

Reply:
Not sure what you mean but the CopyFromString method is removed here.

Reply:
oh nevermind, I expect these tests are now on the dfdatetime side, which is why I don't see them in this PR